tools/default_software.py

```diff
@@ -0,0 +1,43 @@
+__all__ = ["get_returnn_root"]
+
+import logging
+from typing import Optional
+
+from sisyphus import gs, Path
+
+import i6_core.tools as tools
+
+RETURNN_COMMIT = None
+
+
+def get_returnn_root(returnn_root: Optional[Path]) -> Path:
```

Comment:
I think the whole handling starts to get rather complicated, so lets move it to a separate function. I put it into a separate package, so that we can treat all similar cases (RASR, g2p, ..) consistently.

What do you think about
  1. the idea of creating such functions in a separate package?
  2. this specific function for returnn_root?